src/plugins/LADTable/components/LADRow.vue

```diff
@@ -125,9 +125,13 @@ export default {
 
         this.timestampKey = this.openmct.time.timeSystem().key;
 
-        this.valueMetadata = this.metadata ? this
-            .metadata
-            .valuesForHints(['range'])[0] : undefined;
+        this.valueMetadata = undefined;
+
+        if (this.metadata) {
+            this.valueMetadata = this
+                .metadata
+                .valuesForHints(['range'])[0] || this.firstNonDomainAttribute(this.metadata);
```

Comment:
Do we need to pass in `this.metadata`? I don't think we are or will use `firstNonDomainAttribute` anywhere else. Could just reference `this.metadata` in the method? This is obviously not a blocker to this, so I'll approve, but just curious on your thoughts.

Reply:
Yeah that's a fair question. I think you could make good arguments both ways. 

I tend to do this to improve readability (it tells a developer that this function uses telemetry metadata, and requires it as an input) and reusability (you could copy the function elsewhere without having to make any changes to it because dependencies are passed into it).

On the other hand it could actually serve to increase confusion because a developer might be concerned that the metadata being passed in is coming from elsewhere, and that there are multiple instances of metadata being tracked, when in fact there aren't.

Since `firstNonDomainAttribute` is really a utility function, and not a logical behavior of the Vue component it is a member of, (it just resides here for convenience ) I'm ok with leaving it as is. I suspect there are other cases where this function might be useful.

Reply:
Nice, that makes sense, I was thinking along the same lines and wanted to see how you thought about it. Thanks!